python/src/dist_bundle1_module.i

```diff
@@ -41,6 +41,7 @@
 %include ChiSquare.i
 %include ChiSquareFactory.i
 %include CompositeDistribution.i
+%include CombinationsDistribution.i
```

Comment:
This should be in the experimental module. I fixed it in the commit related to the MarginalUniformOrderStatistics, my bad

Reply:
yes, it would be easier to squash the CombinationsDistribution commits